code/API_definitions/qod-api.yaml

```diff
@@ -329,16 +332,32 @@ components:
       type: object
       required:
         - sessionId
-        - event
+        - eventReport
       properties:
         sessionId:
           $ref: "#/components/schemas/SessionId"
-        event:
+        eventReport:
           $ref: "#/components/schemas/SessionEvent"
     SessionEvent:
-      type: string
-      enum:
-        - SESSION_TERMINATED
+      type: object
+      required:
+        - event
+      properties:
+        event:
+          type: string
+          enum:
+            - QOS_STATUS_CHANGED
```

Comment:
If `type` is added to support future events, and each type of event may have different properties, this should be designed as a polymorphic object defining `type` as a discriminator, or alternatively define a `oneOf` straucture.